### DISCUSSION TITLE
IMAGES FOR WIKI

### DISCUSSION THREAD

**Discussion by Foxlider** [Foxlider/FASTER#57] [Show and tell]
This is the issue to host images for the wiki

> Steam Updater
![image](https://user-images.githubusercontent.com/19773387/95012856-2eecb000-063c-11eb-94fa-150f3133ba42.png)

>Steam Mods
![image](https://user-images.githubusercontent.com/19773387/95012867-4cba1500-063c-11eb-8f1a-58f13880b31b.png)

>local Mods
![image](https://user-images.githubusercontent.com/19773387/95012878-5f344e80-063c-11eb-99e7-0871bb76b747.png)

> Server Status
![image](https://user-images.githubusercontent.com/19773387/95012882-6d826a80-063c-11eb-9110-b66a74dea32d.png)

> Server Status Additional
![image](https://user-images.githubusercontent.com/19773387/95015430-26e93c00-064d-11eb-9c8c-292d202e177f.png)

> Server Status Ex
![image](https://user-images.githubusercontent.com/19773387/95015745-faceba80-064e-11eb-938a-dd72fcfec6cd.png)

> Profile.Profile
![image](https://user-images.githubusercontent.com/19773387/95013019-a8d16900-063d-11eb-8268-9895684c3446.png)

> Profile.Missions
![image](https://user-images.githubusercontent.com/19773387/95013028-b71f8500-063d-11eb-9b56-03b0ac4708a3.png)

> Profile.Missions override
![image](https://user-images.githubusercontent.com/19773387/95013092-285f3800-063e-11eb-871f-16e1bded1d85.png)

> Profile.Server
![image](https://user-images.githubusercontent.com/19773387/95013037-d0c0cc80-063d-11eb-8ca1-ba3f445e5461.png)

> profile.Server output
![image](https://user-images.githubusercontent.com/19773387/95013066-edf59b00-063d-11eb-8738-80f2439427c7.png)

> RightClick options
![image](https://user-images.githubusercontent.com/19773387/95013074-02399800-063e-11eb-9d0a-c56e1b135053.png)

> Profile.Performance
![image](https://user-images.githubusercontent.com/19773387/95013049-ddddbb80-063d-11eb-9735-fa7009dcee1b.png)

> Profile perf presets
![image](https://user-images.githubusercontent.com/19773387/95013133-7aa05900-063e-11eb-8261-07a6c5bbe5a8.png)

**Comment by Foxlider** [comment]
![Drawing1](https://user-images.githubusercontent.com/19773387/147383835-630a350b-a4c0-4a46-bf4b-db20e621808c.png)
![image](https://user-images.githubusercontent.com/19773387/147383857-ab63ab77-aeef-42ad-8b51-e0708096fbf9.png)
![image](https://user-images.githubusercontent.com/19773387/147383875-3eb63a32-ada4-4faa-a283-e4ae2b023fc9.png)

**Comment by Foxlider** [comment]
![image](https://user-images.githubusercontent.com/19773387/147384230-3da8cb8d-4a3a-4ae2-90f4-3fc90a3954ad.png)

**Comment by Foxlider** [comment]
![standards](https://user-images.githubusercontent.com/19773387/159466938-81088002-bde0-4a7d-8a29-1c73f9ada497.gif)

**Comment by jupster** [comment]
Profile.Performance
![Profile.Performance](https://github.com/Foxlider/FASTER/assets/78136088/ba14994a-c1e9-4268-80b8-52c25a7c05ff)

Profile.Profile
![profile mods](https://github.com/Foxlider/FASTER/assets/78136088/2488268d-9a55-44cc-9010-c306c5056a33)

**Comment by jupster** [comment]
Updater
![Updater](https://github.com/Foxlider/FASTER/assets/78136088/e4e191ef-9477-4400-ab87-2f266e26db73)

Mods
![Mods](https://github.com/Foxlider/FASTER/assets/78136088/2efd678b-0935-4cf5-b3f2-62663ef691a2)

Profile
![Profile.Profile](https://github.com/Foxlider/FASTER/assets/78136088/7ed05316-2e7c-471e-8c7a-495b0a4afcb8)

Missions
![Profile.Missions](https://github.com/Foxlider/FASTER/assets/78136088/834066fb-1db7-4751-9b45-d137069b90e6)

Server
![Profile.Server](https://github.com/Foxlider/FASTER/assets/78136088/705c9a66-415c-4d4c-a845-da522b7833ae)

Server Advanced
![Profile.Server.Advanced](https://github.com/Foxlider/FASTER/assets/78136088/5b13cd38-d339-4961-b1bd-8a6fbb448677)

Performance
![Profile.Performance](https://github.com/Foxlider/FASTER/assets/78136088/f8390279-8e59-4340-8ad6-b07ac159cda3)
